index.js

```diff
@@ -805,159 +818,205 @@ function buildValue (location, input) {
     type = 'string'
   }
 
-  switch (type) {
-    case 'null':
-      code += 'json += serializer.asNull()'
-      break
-    case 'string': {
-      funcName = nullable ? 'serializer.asStringNullable.bind(serializer)' : 'serializer.asString.bind(serializer)'
-      code += `json += ${funcName}(${input})`
-      break
+  if ('const' in schema) {
+    const stringifiedSchema = JSON.stringify(schema.const)
```

Comment:
I am not so convinced by this, as it is not deterministic and would break probably on circular references and bigint.

Reply:
So probably returns false if you set const: { a: 1, b: 2 } then it would return false on { b: 2, a: 1} 

Maybe we need here rfdc instead of stringification for objects 

Also what happens when we provide a Date.

E.g. 

const: { }

and JSON.stringify on Date is also returning { }

So probably it would pass on Dates.

Reply:
sry, my mistake JSON.stringify(Date) returns the ISODateTime. But still a potential pitfall.

Reply:
Agree on `rfdc` to remove any `__proto__`

We know that schemas should be treated as code, but adding this `const` check gives to the user the possibility to inject bad code